site/page.js

```diff
@@ -34,7 +38,9 @@ class Page extends React.Component {
 }
 
 Page.propTypes = {
-  children: PropTypes.node.isRequired
+  children: PropTypes.node.isRequired,
+  // Should start and end with a slash.
```

Comment:
I thought a comment would be sufficient for this site, but for more high-pressure sites of course we'll want to automate any rules like this.